lib/ext_json.js

```diff
@@ -146,6 +146,30 @@ const stringify = function(value, reducer, indents, options) {
   return JSON.stringify(doc, reducer, indents);
 };
 
+/**
+ * serializes BSON to JSON
```

Comment:
This is still a misnomer. What is going on here is that we are taking a JavaScript representation (object) of BSON types and converting them to a JavaScript representation of the extended JSON generated from the source object. BSON === binary data, EJSON === string data, input and output of this function are JavaScript `Object`s